content/blogposts/2024/cody-vscode-1-16-0-release.md

```diff
@@ -0,0 +1,108 @@
+
+This hotkey makes it easier to ask general coding questions or questions where you _only_ want Cody to use `@-mentioned` context. 
+
+
+## We’ve upgraded Cody Pro users on Claude 2 to Claude 3 Sonnet
+
+Claude 2 model support has been removed in favor of the newer Claude 3 models on Cody Free and Pro tiers. If you’re a Cody Pro user with Claude 2 as your default model, you’ll automatically be upgraded to Claude 3 Sonnet when you update your extension. All chats that used Claude 2 will also be upgraded to Claude 3.
+
+
+## Experimental features 🧪
+
+We’re shipping two experimental features this week. These projects are very fresh and are less stable than Cody’s production-ready features. We don’t recommend adding them to your daily workflow yet, but if you’re interested in trying out the newest features, we welcome your feedback!
+
+
+### Cody CLI
+
+You can now trigger chat from the command line. You can ask general questions just like you would in your IDE, and Enterprise users can also ask questions that include codebase context. 
+
+<Video 
+  source={{
+    mp4: 'blog/cody-vscode-1-16-release/cody-cli-experimental'
+  }}
+  loop={true}
+  title="Using Cody CLI to learn how to convert input.mov to output.mp4"
+  caption="Using Cody CLI to learn how to convert input.mov to output.mp4"
+/>
+
+The CLI is only intended for human interactive usage. Please [read our thread in the community forum](https://community.sourcegraph.com/t/feedback-on-cody-cli-experimental-feature/78) for known issues and usage limitations before [getting started](https://github.com/sourcegraph/cody/blob/main/cli/README.md). You can also find installation instructions in the thread, and if you have feedback, we’d love for you to share it there.
+
+
+### ContextMentionProvider API: BYO context sources
+
+We’ve exposed a new ContextMentionProvider API. You can use it to build new context sources that you can `@-mention` in chat. You can define `triggerPrefixes`, like `@npm:`, then use those prefixes to select context to pass to Cody.
+
+This API is off by default and only enabled when you set `cody.experimental.noodle` to `true` in VS Code settings. Note that this API and the context provider implementation are subject to change.
+
+[Read the pull request](https://github.com/sourcegraph/cody/pull/3883) for more details.
+
+
+## Changelog
+
+See the [changelog](https://github.com/sourcegraph/cody/releases/tag/vscode-v1.14.0) and [GitHub releases](https://github.com/sourcegraph/cody/releases) for a complete list of changes.
+
+
+## Thank you
+
+Cody wouldn’t be what it is without our amazing contributors 💖 A big thank you to everyone who contributed, filed issues, and sent us feedback.
+
+We value your feedback in our [support forum](https://community.sourcegraph.com/), [Discord](https://discord.com/servers/sourcegraph-969688426372825169) and [GitHub](https://github.com/sourcegraph/cody/discussions). Happy Codying!
```

Comment:
Remove GitHub Discussions. We no longer use it.

Reply:
Fixed!